catalog/spatial/geocoding/spatial-geocoding-query/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneIndex.java

```diff
@@ -213,10 +211,10 @@ query searching for any of the terms in the query string (the OR query). We take
         return new CustomScoreQuery(disjunctionMaxQuery, boostQuery);
     }
 
-    protected List<NearbyLocation> doGetNearestCities(final Metacard metacard, final int radiusInKm,
+    protected List<NearbyLocation> doGetNearestCities(final String location, final int radiusInKm,
             final int maxResults, final Directory directory) {
-        if (metacard == null) {
-            throw new IllegalArgumentException("metacard must not be null.");
+        if (location == null) {
```

Comment:
The point of the test is to pass in a negative value for max_results and to verify that the IllegalArgumentException is thrown.  So the first 2 parameters just need to pass validation, the actual values are irrelevant.